java/flight/flight-integration-tests/pom.xml

```diff
@@ -59,6 +59,11 @@
             <groupId>org.slf4j</groupId>
             <artifactId>slf4j-api</artifactId>
         </dependency>
+        <dependency>
+            <groupId>joda-time</groupId>
+            <artifactId>joda-time</artifactId>
+            <version>2.10.14</version>
+        </dependency>
```

Comment:
Why is this needed in the integration tests module?

Reply:
Removed.